client/src/components/internship/CreateInternship.vue

```diff
@@ -201,11 +190,11 @@
               </select>
             </div>
             <div class="col">
-              <label for="newCompanySize">Größe</label>
+              <label for="newCompanySize">{{ $t("company.size") }}</label>
               <select v-model="newCompanySize"
                       id="newCompanySize"
                       class="form-control">
-                <option value="">Bitte auswählen</option>
+                <option value="">{{ $t("company.select") }}</option>
                 <option value="big">mehr als 250 Angestellte</option>
                 <option value="medium">weniger als 250 Angestellte</option>
                 <option value="small">weniger als 50 Angestellte</option>
```

Comment:
Vielleicht auch localizen? (more than 250 employees, less than 250 employees, less than 50 employees)

Reply:
das hab ich erstmal gelassen weil das is ja n enum im backend, ich dachte vlt kann ich das gleich irgendwie damit synchronisieren?